apis/core.d.ts

```diff
@@ -11,21 +11,23 @@ type Intersect<T extends readonly unknown[]> = T extends [infer Head, ...infer T
  * Base class for linked Associations from reflected models.
  * @see [capire](https://cap.cloud.sap/docs/node.js/cds-reflect#cds-Association)
  */
-export type Association = new(_?:object) => LinkedAssociation
+export const Association: new(_?:object) => LinkedAssociation
+
 /**
  * Base class for linked Compositions from reflected models.
  * @see [capire](https://cap.cloud.sap/docs/node.js/cds-reflect#cds-Association)
  */
-export type Composition = new(_?:object) => LinkedAssociation
+export const Composition: new(_?:object) => LinkedAssociation
+
 /**
  * Base class for linked entities from reflected models.
  * @see [capire](https://cap.cloud.sap/docs/node.js/cds-reflect#cds-entity)
  */
-export type entity = new(_?:object) => LinkedEntity
-export type event = new(_?:object) => linked & csn.struct
-export type type = new(_?:object) => linked & csn.type
-export type array = new(_?:object) => linked & csn.type
-export type struct = new(_?:object) => linked & csn.struct
+export const entity: new(_?:object) => LinkedEntity
+export const event: new(_?:object) => linked & csn.struct
+export const type: new(_?:object) => linked & csn.type
+export const array: new(_?:object) => linked & csn.type
+export const struct: new(_?:object) => linked & csn.struct
```

Comment:
These are the main fixes of the reflected types.  `export type` didn't expose them in the `cds` facade object, but `export const` does.

Reply:
I believe they should be available as types. But _only_ as types.

As such, the following should work with the original `export type ...` variant:

```ts
let x: cds.struct  // .struct is auto completed and x now offers intellisense of the corresponding type
```

Reply:
I am addressing this issue in https://github.com/cap-js/cds-types/pull/20